build_runner/lib/src/package_graph/package_graph.dart

```diff
@@ -191,20 +191,20 @@ class PackageNode {
   ///
   /// Only the files matching these patterns will end up in the asset graph,
   /// unless they also match a pattern in [excludes].
-  final List<String> _includes;
-  Iterable<String> get includes => _includes;
+  @deprecated
+  final Iterable<String> includes;
```

Comment:
Are these used at all any more?

Reply:
They are part of the public API.

There is very likely not any other code relying on this, but removing it completely would be a breaking change.